src/main/java/org/sonatype/nexus/ci/nxrm/NexusPublisherBuildStep.groovy

```diff
@@ -71,20 +86,34 @@ class NexusPublisherBuildStep
       true
     }
 
+    @Override
     FormValidation doCheckNexusInstanceId(@QueryParameter String value) {
       NxrmUtil.doCheckNexusInstanceId(value)
     }
 
+    @Override
     ListBoxModel doFillNexusInstanceIdItems() {
       NxrmUtil.doFillNexusInstanceIdItems()
     }
 
+    @Override
     FormValidation doCheckNexusRepositoryId(@QueryParameter String value) {
       NxrmUtil.doCheckNexusRepositoryId(value)
     }
 
+    @Override
     ListBoxModel doFillNexusRepositoryIdItems(@QueryParameter String nexusInstanceId) {
       NxrmUtil.doFillNexusRepositoryIdItems(nexusInstanceId)
     }
+
+    @SuppressWarnings('UnusedMethodParameter')
+    FormValidation doCheckTagName(@QueryParameter String tagName) {
```

Comment:
Q: Do we need these validation methods given we're not actually doing any checks on the tag name (as it's optional)?